docs/development/affiliated-packages.rst

```diff
@@ -404,9 +404,9 @@ files manually`_ section since this explains what many of the files do.
    replace ``packagename`` with the name of your package.
 
 #. If you want the documentation for your project to be hosted by
-   `ReadTheDocs <https://readthedocs.org>`_, then you need to setup an
+   `ReadTheDocs <https://readthedocs.io>`_, then you need to setup an
```

Comment:
I think this one needs to stay the same; the `.io` is specifically for hosted projects, not readthedocs itself (though it redirects correctly).